### DISCUSSION TITLE
Python SQLite3  --enable-loadable-sqlite-extensions

### DISCUSSION THREAD

**Discussion by rsforbes** [devcontainers/community#57] [Q&A]
I have a development requirement to use SQLite extensions. I'm not certain how to complete this within the context of a dev container.

The Python [documentation](https://docs.python.org/3/library/sqlite3.html#sqlite3.Connection.enable_load_extension) states the following

```
Note The sqlite3 module is not built with loadable extension support by default, because some
platforms (notably macOS) have SQLite libraries which are compiled without this feature. To get 
loadable extension support, you must pass the --enable-loadable-sqlite-extensions option to 
configure.
```

Python configure option [--enable-loadable-sqlite-extensions](https://docs.python.org/3/using/configure.html#cmdoption-enable-loadable-sqlite-extensions)

My devcontainer.json (features/image)
```
	"features": {
		"ghcr.io/devcontainers/features/python:latest": {
			"version": "3.11"
		},
		"ghcr.io/devcontainers-contrib/features/poetry:2": {}
	},
	"image": "mcr.microsoft.com/devcontainers/base:debian",
```

**Comment by bamurtaugh** [comment]
Thanks for sharing this @rsforbes! I'd also recommend joining and sharing this in our dev container community Slack channel, as a broader set of folks from the dev container community may have expertise and thoughts on this: https://aka.ms/dev-container-community.
